plugins/node/opentelemetry-instrumentation-mysql2/test/mysql.test.ts

```diff
@@ -49,7 +50,7 @@ interface Result extends mysqlTypes.RowDataPacket {
   solution: number;
 }
 
-describe('mysql2@2.x', () => {
+describe('mysql2@2.x includeValuesInDbStatement=true (old behaviour)', () => {
```

Comment:
The text "old behaviour" is relevant now, but it will probably stay in the code base for a long time and will be less clear without the present context.